### PR TITLE
magento/merchdocs#1089: Add RMA how-to instructions to Returns section

### PR DESCRIPTION
**Pull request by ZhovnirMaria**
## Purpose of this pull request

This pull request adds an instruction for how to create returns requests from the Admin side. 

## Magento release version

- [x] 2.4.x

   Specify a patch release number, if applicable:

- [ ] 2.3.x

   Specify a patch release number, if applicable:

## Product editions

Is this update specific to a product edition?

- [x] Magento Open Source only
- [x] Magento Commerce only

Is this update specific to an installed feature extension

- [x] B2B extension
- [x] Other feature set, please specify:

## Affected documentation pages

https://docs.magento.com/user-guide/sales/returns.html

### CODE REVIEW

File: src/sales/returns.md [hunk 1]
```diff
@@ -8,11 +8,59 @@ redirect_from:
...
 
 RMAs can be issued for simple, grouped, configurable, and bundle product types. However, RMAs are not available for virtual products, downloadable products, and gift cards.
 
+## Create a Return Request
+
+1. To create new return request, go to **Sales** > **Returns** > Click <span class="btn">New Return Request</span>.
+
+1. Click on an order with a `Complete` status to create a return request.
+
+    ![]({% link images/images/new-return.png %}){: .zoom}
```
Comment (dobooth) [inline src/sales/returns.md:22]: We can remove this image.

File: src/sales/returns.md [hunk 2]
```diff
@@ -8,11 +8,59 @@ redirect_from:
...
+    _New Return Page_
+
+1. Select the **Return Items** tab under the **Return Information** section.
+
+1. To add items to return, click <span class="btn">Add Items</span>.
+
+1. Select the checkbox for the required product and click <span class="btn">Add Selected Product to returns</span>.
+
+    ![]({% link images/images/return-item.png %}){: .zoom}
```
Comment (dobooth) [inline src/sales/returns.md:31]: And this one can go as well.

**Review by dobooth — CHANGES_REQUESTED** [review]

File: src/sales/returns.md (hunk 2 above)
Comment (ZhovnirMaria) [reply src/sales/returns.md]: @dobooth 
Done, thanks!

File: src/sales/returns.md [hunk 3]
```diff
@@ -8,11 +8,53 @@ redirect_from:
 
 A returned merchandise authorization (RMA) can be granted to customers who request to return an item for replacement or refund. Typically, the customer contacts the merchant to request a refund. If approved, a unique RMA number is assigned to identify the returned product. In the configuration, you can either enable RMA for all products or allow RMA for only certain products. The _Returns_ grid lists the current returned merchandise requests (RMAs) and is used to enter new return requests.
 
-![]({% link images/images/sales-returns.png %}){: .zoom}
+![]({% link images/images/sales-return.png %}){: .zoom}
 _Returns grid_
 
 RMAs can be issued for simple, grouped, configurable, and bundle product types. However, RMAs are not available for virtual products, downloadable products, and gift cards.
 
+## Create a Return Request
```
Comment (leslietilling) [inline src/sales/returns.md:16]: Sentence case for headings
```suggestion
## Create a return request
```

File: src/sales/returns.md [hunk 4]
```diff
@@ -8,11 +8,53 @@ redirect_from:
...
+
+1. To create new return request, go to **Sales** > **Returns** > Click <span class="btn">New Return Request</span>.
```
Comment (leslietilling) [inline src/sales/returns.md:18]:
```suggestion
1. On the _Admin_ sidebar, go to **Sales** > **Returns**.

1.  Click <span class="btn">New Return Request</span>.
```

File: src/sales/returns.md [hunk 5]
```diff
@@ -8,11 +8,53 @@ redirect_from:
...
+
+1. Click on an order with a `Complete` status to create a return request.
```
Comment (leslietilling) [inline src/sales/returns.md:20]:
```suggestion
1. Click an order with a `Complete` status to create a return request.
```

File: src/sales/returns.md [hunk 6]
```diff
@@ -8,11 +8,53 @@ redirect_from:
...
+
+1. Select the **Return Items** tab under the **Return Information** section.
```
Comment (leslietilling) [inline src/sales/returns.md:22]:
```suggestion
1. Under the _Return Information_ section, select the **Return Items** tab.
```

File: src/sales/returns.md [hunk 7]
```diff
@@ -8,11 +8,53 @@ redirect_from:
...
+
+1. To add items to return, click <span class="btn">Add Items</span>.
+
+1. Select the checkbox for the required product and click <span class="btn">Add Selected Product to returns</span>.
+
+1. Fill in the field **Requested** with the number of items you want to return.
```
Comment (leslietilling) [inline src/sales/returns.md:28]:
```suggestion
1. For **Requested**, enter the number of items to be returned.
```

File: src/sales/returns.md [hunk 8]
```diff
@@ -8,11 +8,53 @@ redirect_from:
...
+
+1. Sets **Return Reason** to one of the following:
```
Comment (leslietilling) [inline src/sales/returns.md:30]:
```suggestion
1. Set **Return Reason** to one of the following:
```

File: src/sales/returns.md [hunk 9]
```diff
@@ -8,11 +8,53 @@ redirect_from:
...
+
+    - Wrong Color
+    - Wrong Size
+    - Out of Service
+    - Other
+
+    If the reason for the return is different from those listed in the **Reason for Return** field, you can enter your own in the field that appears below when you select `Other` option.
```
Comment (leslietilling) [inline src/sales/returns.md:37]:
```suggestion
    If the reason for the return is different from those listed, you can enter your own if you select the `Other` option.
```

File: src/sales/returns.md [hunk 10]
```diff
@@ -8,11 +8,53 @@ redirect_from:
...
+
+1. Sets **Item Condition** to one of the following:
```
Comment (leslietilling) [inline src/sales/returns.md:39]:
```suggestion
1. Set **Item Condition** to one of the following:
```

File: src/sales/returns.md [hunk 11]
```diff
@@ -8,11 +8,53 @@ redirect_from:
...
+
+    - Unopened
+    - Opened
+    - Damaged
+
+1. Sets **Resolution** to one of the following:
```
Comment (leslietilling) [inline src/sales/returns.md:45]:
```suggestion
1. Set **Resolution** to one of the following:
```

**Review by leslietilling — COMMENTED** [review]
Edits for grammar and style.

**Comment by ZhovnirMaria** [comment]
@leslietilling 
Done, thank you!

**Review by leslietilling — APPROVED** [review]

**Review by dobooth — APPROVED** [review]

**Comment by dobooth** [comment]
@magento import code to magento-devdocs/merchdocs

**Comment by magento-engcom-team** [comment]
@dobooth the branch with code successfully imported into`magento-devdocs/merchdocs` repository. Branch name: `imported-magento-merchdocs-1183`.

**Pull request merged**